src/handler/SpecifyDependenciesStep.ts

```diff
@@ -63,7 +63,19 @@ export class SpecifyDependenciesStep implements IStep {
                     }),
                     pickBox.onDidHide(() => {
                         reject(new OperationCanceledError("Canceled on dependency selection."));
-                    })
+                    }),
+                    pickBox.onDidTriggerItemButton(e => {
+                        let href: string | undefined = (e.button as any).href;
+                        if (href) {
```

Comment:
Is it possible to check if the href is available or not at the stage of adding buttons to the quickpick list?

Reply:
checking `href` at the adding button stage. Here simply consume the link without checking, because it's dedicated for spring starters.